README.md

```diff
@@ -1,30 +1,34 @@
-<img src="https://static.scarf.sh/a.png?x-pxid=be2d8a11-9712-4c1d-9963-580b2d4fb133" />
+<img src="https://static.scarf.sh/a.png?x-pxid=be2d8a11-9712-4c1d-9963-580b2d4fb133" alt="Scarf analytics tracking pixel" />
```

Comment:
We probably want to make this an `alt=""` to avoid it being read

Reply:
@crutchcorn Thanks for the feedback!